src/schema/slack/functions/_scripts/src/write_function_files.ts

```diff
@@ -30,26 +32,41 @@ async function main() {
       const filename = `../${functionRecord.callback_id}.ts`;
       const testFilename = `../${functionRecord.callback_id}_test.ts`;
 
-      const templateString = SlackFunctionTemplate(functionRecord);
-      const templateTestString = SlackTestFunctionTemplate(
+      const templateString = _internals.SlackFunctionTemplate(functionRecord);
+      const templateTestString = _internals.SlackTestFunctionTemplate(
         functionRecord,
       );
 
-      await Deno.writeTextFile(filename, templateString);
-      await Deno.writeTextFile(testFilename, templateTestString);
+      await _internals.writeTextFile(filename, templateString);
+      await _internals.writeTextFile(testFilename, templateTestString);
     }),
   );
 
   console.log(
     `Generated ${slackFunctions.length} Slack functions with their unit tests`,
   );
 
-  const modString = SlackFunctionModTemplate(slackFunctions);
+  const modString = _internals.SlackFunctionModTemplate(slackFunctions);
 
-  await Deno.writeTextFile("../mod.ts", modString);
+  await _internals.writeTextFile("../mod.ts", modString);
   console.log("Updated functions module export");
 }
 
+export const _internals = {
```

Comment:
Nice! I've been wanting to try this pattern in our deno projects for a loooong time!